server/models/questions/getEditorQuestions.js

```diff
@@ -76,10 +76,10 @@ function getEditorQuestions({
       countQuery += `WHERE `
     }
 
-    query += ` qe.editor=? `
-    countQuery += ` qe.editor=? `
-    qArr.push(username)
-    cQArr.push(username)
+    query += ` qe.editor=? AND qe.access=?`
+    countQuery += ` qe.editor=? AND qe.access=?`
+    qArr.push(username, 'write')
+    cQArr.push(username, 'write')
```

Comment:
#### CHANGE

1. If only asking for editor questions than `access` = `write` should also be provided.....however if asking for reader questions no need for specifying `access` ..... as `read` access will be by default